src/svg2pdf.js

```diff
@@ -279,6 +279,12 @@ SOFTWARE.
       _pdf.beginFormObject(bBox[0], bBox[1], bBox[2], bBox[3], tfMatrix);
       renderChildren(node, _pdf.unitMatrix, this, false, false, AttributeState.default());
       _pdf.endFormObject(node.getAttribute("id"));
+    } else if (nodeIs(node, "symbol,svg")) {
+      // create unclipped and untransformed form object
+      var bBox = getUntransformedBBox(node);
```

Comment:
we should probably use a larger bBox here. At least in the svg case, this is the box determined by width/height or the viewBox.